src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticBraceCompletionTests.cs

```diff
@@ -1348,150 +1348,6 @@ public void CurlyBraceFormatting_InsertsCorrectNewLine()
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)]
-        public void CurlyBraceFormattingEditMerge_NewLineInsideFormattingChange()
-        {
-            var originalText = SourceText.From("aaaa");
-
-            // Text should be aa#$aa
-            var insertionTextChange = new TextChange(new TextSpan(2, 0), "#$");
-            var incrementalText = originalText.WithChanges(insertionTextChange);
-
-            // Insertion change is entirely inside the formatting change.
-            // Replace '#$' with 'ccc' to become aacccaa
-            var formattingTextChange = new TextChange(new TextSpan(2, 2), "ccc");
-            incrementalText = incrementalText.WithChanges(formattingTextChange);
-
-            var mergedChanges = MergeFormatChangesIntoNewLineChange(insertionTextChange, ImmutableArray.Create(formattingTextChange));
-            var mergedText = originalText.WithChanges(mergedChanges);
-
-            // Only one edit is returned with the new text.
-            Assert.Equal(new TextChange(new TextSpan(2, 0), "ccc"), mergedChanges.Single());
-            Assert.Equal(incrementalText.ToString(), mergedText.ToString());
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)]
-        public void CurlyBraceFormattingEditMerge_FormattingBeforeAndAfter()
-        {
-            var originalText = SourceText.From("aaaa");
-
-            // Text should be aa#$%aa
-            var insertionTextChange = new TextChange(new TextSpan(2, 0), "#$%");
-            var incrementalText = originalText.WithChanges(insertionTextChange);
-
-            // Multiple changes formatting changes before and after the insertion.
-            // Result should be ccdd$eeff
-            var entirelyBeforeChange = new TextChange(new TextSpan(0, 1), "cc");
-            var partiallyBeforeChange = new TextChange(new TextSpan(1, 2), "dd");
-            var partiallyAfterChange = new TextChange(new TextSpan(4, 2), "ee");
-            var entirelyAfterChange = new TextChange(new TextSpan(6, 1), "ff");
-            var changes = ImmutableArray.Create(entirelyBeforeChange, partiallyBeforeChange, partiallyAfterChange, entirelyAfterChange);
-            incrementalText = incrementalText.WithChanges(changes);
-
-            var mergedChanges = MergeFormatChangesIntoNewLineChange(insertionTextChange, changes);
-            var mergedText = originalText.WithChanges(mergedChanges);
-
-            Assert.Equal(new TextChange(new TextSpan(2, 0), "$"), mergedChanges[0]);
-            Assert.Equal(new TextChange(new TextSpan(0, 1), "cc"), mergedChanges[1]);
-            Assert.Equal(new TextChange(new TextSpan(1, 1), "dd"), mergedChanges[2]);
-            Assert.Equal(new TextChange(new TextSpan(2, 1), "ee"), mergedChanges[3]);
-            Assert.Equal(new TextChange(new TextSpan(3, 1), "ff"), mergedChanges[4]);
-            Assert.Equal(incrementalText.ToString(), mergedText.ToString());
-        }
```

Comment:
consider still having these tests, but potentially move to compiler layer.

Reply:
There's already a bunch of tests for the merge function here - http://sourceroslyn.io/#Microsoft.CodeAnalysis.UnitTests/Text/TextChangeTests.cs,20 - these end up just being dupes.